### PR TITLE
Cleaned up documentation.

### PR DESCRIPTION
**Pull request by pv101**
The CLI section largely duplicated the User Guide -> Running PyFR section, so I have merged, and removed the CLI section. But @WillTrojak do let me know if you think good reason to retain separate CLI docs at this stage.

### CODE REVIEW

File: doc/src/user_guide.rst [hunk 1]
```diff
@@ -37,8 +37,8 @@ The following commands are available from the ``pyfr`` program:
    directory `outdir`.  Multiple solutions can be provided.
    Time-average files can also be partitioned, too.
 
-   For mixed grids it is usually necessary to provide weights for each
-   element type.  Further details can be found in the
+   For mixed grids one must provide weights for each element type.
```
[the remark this answers is not in the document]
Comment (FreddieWitherden) [reply doc/src/user_guide.rst:40]: Yes, some flag is always needed

File: doc/src/plugins/soln-plugin-fluid-force.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
+*******************************
+[soln-plugin-fluidforce-*name*]
+*******************************
+
+Periodically integrates the pressure and viscous stress on the boundary
+labelled ``name`` and writes out the resulting force and moment (if requested)
+vectors to a CSV file. Parameterised with
```
Comment (FreddieWitherden) [inline doc/src/plugins/soln-plugin-fluid-force.rst:7]: Add a paragraph describing the moment stuff.

File: doc/src/plugins/soln-plugin-tavg.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
+******************
+[soln-plugin-tavg]
+******************
+
+Time average quantities. Parameterised with
```
Comment (FreddieWitherden) [inline doc/src/plugins/soln-plugin-tavg.rst:5]: Add a couple of paragraphs with more detail around usage (windowing modes, getting fluctuations, ...)

**Comment by FreddieWitherden** [comment]
What still remains to be done here?

**Comment by pv101** [comment]
> What still remains to be done here?

Add something about CLI interface to Ascent plugin - did you have a few notes on that you can share?

**Comment by pv101** [comment]
I have added Ascent CLI docs. For me this is now good to merge. Thoughts?

**Pull request merged**